### ISSUE TITLE
No ID is given to contact when adding it using api

### ISSUE THREAD

**Issue by usbulat** [moira-alert/moira#86]
Hi,

v2.2.0-1
When using api to create contact with certain ID
```
curl -s -X PUT http://localhost:80/api/contact -d @moira_objects/contact.json
{"type":"mail","value":"user@gmail.com"}
```
the contact is created, but without ID.

contact.json file is:
```
{
  "type": "mail",
  "value": "user@gmail.com",
  "id": "4f8bb896-a213-443e-bab7-3105763910ae",
  "user": ""
}
```

**Comment by titusjaka** [comment]
Hi Bulat,

Thank you for noticing this issue. It was a bug in API, we've already fixed it in last commit. This changes will be available in the coming release 2.3.

If you need this behavior on your daily basis right now, you can download sources and build it by yourself: [Build Moira Microservices](https://moira.readthedocs.io/en/latest/installation/manual.html#build-moira-microservices).

**Comment by titusjaka** [comment]
You can also install the latest version of Moira using docker: [Docker installation](https://moira.readthedocs.io/en/latest/installation/docker.html)
